specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+            "x-nullable": false,
+            "description": "In case an error has occurred in the server",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PageResultOfReviewDto": {
+      "type": "object",
+      "required": [
+        "count"
+      ],
+      "properties": {
+        "items": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReviewDto"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "nextPageLink": {
+          "type": "string",
+          "format": "uri"
+        }
+      }
+    },
+    "ReviewDto": {
+      "type": "object",
+      "required": [
+        "rating",
+        "createdDateTime",
+        "lastModifiedDateTime",
+        "commentsCount",
+        "helpfulCount"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "uniqueProductId": {
+          "type": "string"
+        },
+        "rating": {
+          "type": "number",
+          "format": "decimal"
+        },
+        "title": {
+          "type": "string"
+        },
+        "content": {
+          "type": "string"
+        },
+        "source": {
+          "type": "string"
+        },
+        "reviewerName": {
+          "type": "string"
+        },
+        "externalReviewUrl": {
+          "type": "string"
+        },
+        "isvReply": {
+          "$ref": "#/definitions/IsvReplyDto"
+        },
+        "createdDateTime": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "lastModifiedDateTime": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "commentsCount": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "helpfulCount": {
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "IsvReplyDto": {
+      "type": "object",
+      "required": [
+        "repliedDateTime"
+      ],
+      "properties": {
+        "content": {
+          "type": "string"
+        },
+        "repliedDateTime": {
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "UserMetadataDto": {
+      "type": "object",
+      "required": [
+        "isEligible"
+      ],
+      "properties": {
+        "review": {
+          "$ref": "#/definitions/ReviewDto"
+        },
+        "helpfulReviewIds": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "commentReviewIds": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "isEligible": {
+          "$ref": "#/definitions/EligibilityStatus"
+        }
+      }
+    },
+    "EligibilityStatus": {
+      "type": "string",
+      "description": "",
+      "x-enumNames": [
```

Comment:
I'm not familiar with `x-enumNames`, this should be an extensible enum with `x-ms-enum`, see https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/creating-swagger.md#enums-with-x-ms-enum